p2pserver/common/p2p_common.go

```diff
@@ -30,15 +30,20 @@ const (
 
 //msg cmd const
 const (
-	MSG_CMD_LEN      = 12  //msg type length in byte
-	CMD_OFFSET       = 4   //cmd type offet in msg hdr
-	CHECKSUM_LEN     = 4   //checksum length in byte
-	HASH_LEN         = 32  // hash length in byte
-	MSG_HDR_LEN      = 24  //msg hdr length in byte
-	MAX_BLK_HDR_CNT  = 500 //hdr count once when sync header
-	MAX_INV_HDR_CNT  = 500 //inventory count once when req inv
-	MAX_REQ_BLK_ONCE = 16  //req blk count once from one peer when sync blk
+	MSG_CMD_LEN      = 12                //msg type length in byte
+	CMD_OFFSET       = 4                 //cmd type offet in msg hdr
+	CHECKSUM_LEN     = 4                 //checksum length in byte
+	HASH_LEN         = 32                // hash length in byte
+	MSG_HDR_LEN      = 24                //msg hdr length in byte
+	MAX_BLK_HDR_CNT  = 500               //hdr count once when sync header
+	MAX_INV_HDR_CNT  = 500               //inventory count once when req inv
+	MAX_REQ_BLK_ONCE = 16                //req blk count once from one peer when sync blk
+	MAX_MSG_LEN      = 100 * 1024 * 1024 // the maximum message length
```

Comment:
consider to use a smaller value? maybe 30M?

Reply:
If the maximum transaction count is 600,000, each transaction is 200B。 600000 * 200B = 114M。  30M may used for at most 150,000 transaction. is it ok?

Reply:
have comfirmed with @luodanwg . 30M is large enough for mainnet. so be conservative with this.

Reply:
the default maximum block transaction count is 60000 actually, 600000 is written by mistake.

Reply:
ok, the commit has updated. please review it, thanks